scripts/main/battle_taunt.lua

```diff
@@ -0,0 +1,60 @@
+function Mod:initBattleTaunt()
+    self.taunt_cooldown = 0
+    self.state_blacklist = {
+        "DEFENDINGBEGIN",
+        "DEFENDING",
```

Comment:
IMO it would be nice if you note that taunting is handled by our Soul hook in DEFENDING

Reply:
Also I would like to remind that this is still in the Mod namespace so I suggest to prefix the variable name

Reply:
i was thinking about it after the pr was merged, whoops